### DISCUSSION TITLE
blog/implementing-gitops-and-canary-deployment-with-argo-project-and-istio/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#351] [Announcements]
# blog/implementing-gitops-and-canary-deployment-with-argo-project-and-istio/

本文讨论如何使用 Kubernetes Deployment、Argo 项目和 Istio 来实现 GitOps 和金丝雀部署。

https://jimmysong.io/blog/implementing-gitops-and-canary-deployment-with-argo-project-and-istio/

**Comment by cnaiplus** [comment]
写得非常好！
